Documentation/guides/messages/xa0125.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Xamarin.Android warning XA0125
+description: XA0125 warning code
+ms.date: 09/28/2020
+---
+# Xamarin.Android warning XA0125
+
+## Issue
+
+Using symbol files with the `.mdb` extension or
```

Comment:
This description doesn't quite go far enough; it can be easily interpreted as only referring to the "current" project.  It should explicitly call out that non-portable PDB symbols of *external dependencies* such as NuGet packages will *also* no longer be supported.

Reply:
I added a couple sentences:
```
...
This also includes NuGet packages or any library that distributes `.mdb` or non-portable `.pdb` files.
...
Notify the library author if you are encountering the `XA0125` warning from a NuGet package.
```

Reply:
Ooo. This raises a potentially interesting question about the wording of the message.

I think Jonathan Dick had some comments about providing more guidance to NuGet package authors for the `error XA0121: Assembly '{0}' is using '[assembly: {1}]', which is no longer supported. Use a newer version of this NuGet package or notify the library author.` messages, which were in some ways similar deprecations in the context of NuGet packages. As I'm writing that down though, I'm remembering that part of the challenge for those messages was that the new approach for the NuGet package authors wasn't just changing an MSBuild property but instead changing how external libraries were packaged in the NuGet package and adding a `.targets` file to the NuGet package. So in fact, in this new case, the current message is probably almost all set.

One little question is whether it might be worth adding something similar to the last sentence from those previous messages?

```markdown
If this file comes from a NuGet package, update to a newer version of
the NuGet package or notify the library author.
```

Reply:
Ooo. This raises a potentially interesting question about the wording of the message.

I think Jonathan Dick had some comments about providing more guidance to NuGet package authors for the `error XA0121: Assembly '{0}' is using '[assembly: {1}]', which is no longer supported. Use a newer version of this NuGet package or notify the library author.` messages, which were in some ways similar deprecations in the context of NuGet packages. As I'm writing that down though, I'm remembering that part of the challenge for those messages was that the new approach for the NuGet package authors wasn't just changing an MSBuild property but instead changing how external libraries were packaged in the NuGet package and adding a `.targets` file to the NuGet package. So in fact, in this new case, the current message is probably almost all set.

One little question is whether it might be worth adding something similar to the last sentence from those previous messages?

```markdown
If this file comes from a NuGet package, update to a newer version of
the NuGet package or notify the library author.
```

Reply:
👍  I updated some of the wording.

Reply:
Next I have a "rendering" concern: look at the formatted Markdown preview: https://github.com/xamarin/xamarin-android/blob/5934f47a2e869c109a7703103b39949f252d1e2e/Documentation/guides/messages/xa0125.md

The error message requires horizontal scrolling to view.  *Lots* of horizontal scrolling.

That might just be GitHub.  What about docs.microsoft.com?

https://docs.microsoft.com/en-us/xamarin/android/errors-and-warnings/xa0115

The above docs URL contains error message which is 274 characters "wide":

```
Invalid value 'armeabi' in $(AndroidSupportedAbis). This ABI is no longer supported. Please update your project properties to remove the old value. If the properties page does not show an 'armeabi' checkbox, un-check and re-check one of the other ABIs and save the changes.
```

This PR adds a message which is *424*:

```
'AndroidApp1.pdb' is using a deprecated debug information level. Set the debugging information to Portable in the Visual Studio project property pages or edit the project file in a text editor and set the 'DebugType' MSBuild property to 'portable' to use the newer, cross-platform debug information level. If this file comes from a NuGet package, update to a newer version of the NuGet package or notify the library author.
```

Perhaps it's just how *I* use web browsers, but these messages are "too long".

Can we get some newlines in there? A'la https://github.com/xamarin/xamarin-android/pull/4614#discussion_r471700258 ?

```
warning XA0125: 'AndroidApp1.pdb' is using a deprecated debug information level.
warning XA0125: Set the debugging information to Portable in the Visual Studio project property pages or edit the project file in a text editor and set the 'DebugType' MSBuild property to 'portable' to use the newer, cross-platform debug information level.
warning XA0125: If this file comes from a NuGet package, update to a newer version of the NuGet package or notify the library author.
```

(MSBuild/the IDE would "add" the extra `warning XA0125` bits…)

Reply:
Adding hard line breaks to the message in the _.resx_ file sounds fine to me too for this case. Just to throw out another option, if the message stayed as-is in the _.resx_ file, I think it would also be fine to hard wrap the text within the code block on the documentation page:

````markdown
```
warning XA0125: 'AndroidApp1.pdb' is using a deprecated debug
information level. Set the debugging information to Portable in the
Visual Studio project property pages or edit the project file in a text
editor and set the 'DebugType' MSBuild property to 'portable' to use the
newer, cross-platform debug information level. If this file comes from a
NuGet package, update to a newer version of the NuGet package or notify
the library author.
```
````

(For a small point of comparison, it seems the [NuGet errors and warnings reference](https://docs.microsoft.com/nuget/reference/errors-and-warnings/nu5127) doesn't do anything too fancy for long messages, so I think xamarin-android has some creative latitude to do something a little different.)

Reply:
I made the XA0125 message 3 lines, and wrapped the two docs files. 👍